config/routes.rb

```diff
@@ -4,10 +4,11 @@
   resources :cart, only: [:create]
   resources :freelancers, only: [:index, :show]
   resources :categories, only: [:index, :show]
-  resources :users, only: [:new, :create]
+  resources :users, only: [:new, :create, :show]
```

Comment:
Let's remember to ask @s-espinosa and @icorson3 about whether we need admins to be able to view user account info.